Compiler/test/preprocessor_test.cpp

```diff
@@ -25,7 +25,7 @@ namespace Preprocessor {
 std::vector<AGSString> SplitLines(const AGSString& str)
 {
     std::vector<AGSString> str_lines = str.Split('\n');
-    for (int i = 0; i < str_lines.size(); i++) {
+    for (int i = 0; i < (int) str_lines.size(); i++) {
```

Comment:
Please fix this other way around, make "i" variable be of `size_t` type instead.